modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryServiceImpl.java

```diff
@@ -532,14 +548,42 @@ private void _checkAddObjectEntryPortletResourcePermission(
 			ObjectActionKeys.ADD_OBJECT_ENTRY);
 	}
 
-	private void _checkPermission(String actionId, ObjectEntry objectEntry)
+	private void _checkPermission(
+			String actionId, long objectDefinitionId, ObjectEntry objectEntry)
```

Comment:
I created a private method because there i had the need for both types of calls which appear in this type of scenario.  And since the one with 3 parameters is a public one, is trickier to change

```
        @Override
	public void checkModelResourcePermission(
			long objectDefinitionId, long objectEntryId, String actionId)
		throws PortalException {

		_checkPermission(
			actionId, objectDefinitionId, getObjectEntry(objectEntryId));
	}

	@Override
	public ObjectEntry deleteObjectEntry(long objectEntryId)
		throws PortalException {

		_checkPermission(
			ActionKeys.DELETE,
			objectEntryLocalService.getObjectEntry(objectEntryId));

		return objectEntryLocalService.deleteObjectEntry(objectEntryId);
	}
```

Reply:
I don't see a value in having a method that **only** calls another method. 
It is possible to have just one because you can get the `objectDefinitionId` by `objectEntry.getObjectDefinitionId()`.